---
title: Research in the Tewhey Lab
layout: default
group: research
---

<img class="img-fluid mx-auto d-block" src="/static/img/research_philip_barter_fall_birches.png" alt="Fall Birches by Philip Barter">
Fall Birches by [Philip Barter](http://thebarterarthouse.com/philip-barter)

# Research Areas
---

The past decade has seen a transformational change in our understanding of the human genome and the role it plays in influencing disease risk. Largescale projects such as the Encyclopedia of DNA Elements (ENCODE) have identified tens of thousands of non­coding regions of the genome that influence gene expression. Furthermore, the proliferation of genome wide association studies (GWAS) have identified thousands of loci in the human genome that influence human health. These efforts have showed us the predominant contributors of heritability for complex phenotypes are common polymorphisms that reside within non-coding regions of the genome. However, despite our progress in mapping cis­regulatory elements (CREs) and genetic signatures correlated with disease, very few examples exist that mechanistically link genotypic variation to disease risk. This gap in our understanding is driven by our inability to understand the sequence context of active CREs and their targets. Without this it remains difficult to identify single nucleotide variants that directly modulate gene expression and in turn understanding their role in disease.

*Our lab is focused on identifying and understanding the precise genetic mechanisms influencing human complex traits and risk to disease. To accomplish this our work spans several areas of interest:*

#### Learning the grammar of cis-regulatory elements and their impact on human traits

Our group develops and applies novel high-throughput experimental methods to characterize regulatory elements in the human genome. We have pioneered the application of massively parallel reporter assays (MPRA) as a tool to understand the impact of genetic variation. We have also developed CRISPR-based screening methods to identify regulatory elements and their target genes. Our primary goal is to apply the methods we develop to characterize the impact of non-coding variation on human disease and translate the associations to a mechanistic understanding. To this end we typically collaborate with domain experts for specific diseases with current projects focusing on type 2 diabetes, colorectal cancer, and autoimmune diseases.


#### Generating mouse models that better reflect human disease risk

To understand the role regulatory elements play on disease risk our group uses the mouse as a model organism to capture the full (patho)physiological environment. We believe this level of complexity is critical and not possible using a standard cell culture approach. Our work focuses on characterizing both individual regulatory elements and developing methods to multiplex modifications at scale. Our goal is to develop mouse models that aggregate multiple human risk alleles across many loci. We believe such an approach will create a superior model that is burdened for disease through the same pathways influencing human risk.   

#### Conservation of gene regulation between species

Modification of gene regulatory elements is a significant driver of evolution. While protein coding regions of the mouse are over 85% similar to humans, non-coding elements are conserved at a significantly lower rate. Furthermore, because of our inability to interpret the function of CREs it is difficult to precisely know the extent of functional conservation between species. To aid in modeling human regulatory variation in the mouse, our group is focused on developing improved maps of homologous CREs between human and mouse. Current comparative approaches rely on sequence homology and correlative measures of gene expression such as regions of DNase hypersensitivity and chromatin modifications. While these methods have provided valuable insight, they lack direct quantitative measurements of a CRE’s impact on individual genes and the location of the cis-regulatory modules within the CREs responsible for activity. Our group is focused on overcoming these shortcomings by developing maps of CRE conservation based directly on function. This work provides insights into the evolution of CREs and aids our ability to properly capture the true effect of human risks alleles when modeled within the mouse.

